internal/inputs/generator_data.go

```diff
@@ -227,6 +227,8 @@ func (g *DataGenerator) getSerializer(sim common.Simulator, format string) (seri
 	switch format {
 	case FormatCassandra:
 		ret = &serialize.CassandraSerializer{}
+	case FormatVictoriaMetrics:
+		fallthrough
```

Comment:
it would be better from readability and maintainability PoV writing `ret = &serialize.InfluxSerializer{}` here instead of `fallthrough`